src/mdio/commands/utility.py

```diff
@@ -0,0 +1,187 @@
+
+    import click
+    import click_params
+
+    import mdio
+except SystemError:
+    pass
+
+
+DEFAULT_HELP = """
+MDIO CLI utilities.
+"""
+
+
+@click.group(help=DEFAULT_HELP)
+def cli():
+    """Setup click group."""
+    click.echo("MDIO CLI utilities")
+
+
+@click.command(name="copy")
+@click.option(
+    "-i",
+    "--input-mdio-path",
+    required=True,
+    help="Input mdio path.",
+    type=click.Path(exists=True),
+)
+@click.option(
+    "-o",
+    "--output-mdio-path",
+    required=True,
+    help="Output path or URL to write the mdio dataset.",
+    type=click.STRING,
+)
+@click.option(
+    "-exc",
+    "--excludes",
+    required=False,
+    help="""Data to exclude during copy. i.e. chunked_012. The raw data won’t be
+    copied, but it will create an empty array to be filled. If left blank, it will
+    copy everything.""",
+    type=click.STRING,
+)
+@click.option(
+    "-inc",
+    "--includes",
+    required=False,
+    help="""Data to include during copy. i.e. trace_headers. If this is not
+    specified, and certain data is excluded, it will not copy headers. If you want
+    to preserve headers, specify trace_headers. If left blank, it will copy
+    everything except specified in excludes parameter.""",
+    type=click.STRING,
+)
+@click.option(
+    "-storage",
+    "--storage-options",
+    required=False,
+    help="Custom storage options for cloud backends",
+    type=click_params.JSON,
+)
+@click.option(
+    "-overwrite",
+    "--overwrite",
+    required=False,
+    default=False,
+    help="Flag to overwrite if mdio file if it exists",
+    type=click.BOOL,
+    show_default=True,
+)
+def copy(
+    input_mdio_path,
+    output_mdio_path: str,
+    includes: str = "",
+    excludes: str = "",
+    storage_options: Optional[dict] = None,
+    overwrite: bool = False,
+):
+    """Copy MDIO to MDIO.
+
+    Can also copy with empty data to be filled later. See `excludes`
+    and `includes` parameters.
+
+    More documentation about `excludes` and `includes` can be found
+    in Zarr's documentation in `zarr.convenience.copy_store`.
+
+    Args:
+        input_mdio_path: MDIO reader or accessor instance. Data will be copied from here
+        output_mdio_path: Destination path. Could be any FSSpec mapping.
+        excludes: Data to exclude during copy. i.e. `chunked_012`. The raw data
+            won't be copied, but it will create an empty array to be filled.
+            If left blank, it will copy everything.
+        includes: Data to include during copy. i.e. `trace_headers`. If this is
+            not specified, and certain data is excluded, it will not copy headers.
+            If you want to preserve headers, specify `trace_headers`. If left blank,
+            it will copy everything except specified in `excludes` parameter.
+        storage_options: Storage options for the cloud storage backend.
+            Default is None (will assume anonymous).
+        overwrite: Overwrite destination or not.
```

Comment:
remove args, this gets populated by click automatically. The `help` above in parameter definitions should resemble or match these.